File: Chapter 0/0004.py
word = input()
small = "abcdefghijklmnopqrstuvwxyz"
big = small.upper()
sm,bi = False,False
for i in word:
    if i in small:
        sm = True
    elif i in big:
        bi = True
if sm and bi:
    print("Mix")
elif sm:
    print("All Small Letter")
elif bi:
    print("All Capital Letter")
        

File: C plus/0004.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string word;
    cin >> word ;
    int n = word.length();
    int i = 0;
    int count = 0;
    for (i = 0 ; i < n ; i++){
        if (word[i] == tolower(word[i])){
            count++;
        }
    }
    if(count == n){
        cout << "All Small Letter" << endl;
    }else if (count == 0){
        cout << "All Capital Letter" << endl;
    }else{
        cout << "Mix" << endl;
    }
    
    

    return 0;
}
